src/OpenAI.csproj

```diff
@@ -48,4 +48,8 @@
     <PackageReference Include="System.ClientModel" Version="1.1.0-beta.4" />
     <PackageReference Include="System.Text.Json" Version="8.0.2" />
   </ItemGroup>
+
+  <ItemGroup>
+    <InternalsVisibleTo Include="Azure.AI.OpenAI, PublicKey=0024000004800000940000000602000000240000525341310004000001000100097ad52abbeaa2e1a1982747cc0106534f65cfea6707eaed696a3a63daea80de2512746801a7e47f88e7781e71af960d89ba2e25561f70b0e2dbc93319e0af1961a719ccf5a4d28709b2b57a5d29b7c09dc8d269a490ebe2651c4b6e6738c27c5fb2c02469fe9757f0a3479ac310d6588a50a28d7dd431b907fd325e18b9e8ed" />
+  </ItemGroup>
```

Comment:
```suggestion
  <ItemGroup>
      <ItemGroup>
    <InternalsVisibleTo Include="Azure.AI.OpenAI">
      <PublicKey>0024000004800000940000000602000000240000525341310004000001000100097ad52abbeaa2e1a1982747cc0106534f65cfea6707eaed696a3a63daea80de2512746801a7e47f88e7781e71af960d89ba2e25561f70b0e2dbc93319e0af1961a719ccf5a4d28709b2b57a5d29b7c09dc8d269a490ebe2651c4b6e6738c27c5fb2c02469fe9757f0a3479ac310d6588a50a28d7dd431b907fd325e18b9e8ed</PublicKey>
    </InternalsVisibleTo>
  </ItemGroup>
  </ItemGroup>
```

Reply:
Not sure about the name for the metadata for PublicKey, according to the documentation it's just Key?
![image](https://github.com/openai/openai-dotnet/assets/3002068/3a06718d-6565-4f05-a7e8-d8dab14cfa0f)

Reply:
I tested, both options work

Reply:
I wasn’t sure that Suggestion had the correct formatting (it seems not, judging by the number of spaces), so I made it a separate commit